packages/fast-usdc/test/supports.ts

```diff
@@ -149,9 +155,11 @@ export const commonSetup = async (t: ExecutionContext<any>) => {
   });
   const timer = buildZoeManualTimer(t.log);
   const marshaller = makeFakeBoard().getReadonlyMarshaller();
-  const storage = makeFakeStorageKit('mockChainStorageRoot', {
-    sequence: false,
-  });
+  const storage = makeFakeStorageKit(
+    'fun', // Fast USDC Node
+  );
+  storage.getPureData = path =>
```

Comment:
Optional: It might help to document this a bit. I gather it can't have any remotables or bigints.

Reply:
good point, it should have a jsdoc.

remotables aren't pure data but bigints are. (bigint is the main motivation; otherwise JSON is a good encoding of pure data)